### ISSUE TITLE
[css-flexbox] Flexbox terminology definitions

### ISSUE THREAD

**Issue by gsnedders** [w3c/csswg-drafts#981]
This all focuses on [the terminology](https://drafts.csswg.org/css-flexbox/#box-model) section.

Figure 2 shows the "main size" taking up the entire width of the flex container. This contradicts the definition of "main size" ("A flex item’s width or height, whichever is in the main dimension, is the item’s main size"), insofar as it applies to a flex item rather than a container.

We also talk about the "flex container’s main size" in various places in the spec, cross-referenced to the same above definition of "main size", which is simply nonsense because the term simply doesn't apply.

All of that equally applies to cross size.

We're also inconsistent as to hyphenation. Is it "main size" or "main-size"? Is it "cross size" or "cross-size"?

**tl;dr: plz define what a flex container's main size and cross size are**

**Comment by fantasai** [comment]
It's cross size as a noun, and cross-size as an adjective. Analogously for main size.

**Comment by gsnedders** [comment]
That is so totally not clear.

**Comment by fantasai** [comment]
It's how English punctuation works.

**Comment by tabatkins** [comment]
Yeah, that's standard English conventions for multi-word terms.

We should clarify that "main/cross size" does apply to flex containers as well; it's their size in the appropriate axis.

**Comment by gsnedders** [comment]
LGTM.
